### ISSUE TITLE
No data on chart / area "Average for the week"

### ISSUE THREAD

**Issue by SebMidon** [YunoHost-Apps/aeneria_ynh#26]
### Describe the bug
No data on chart / area "Average for the week"

### Context
- Instance of Aeneria installed on a Raspberry Pi 3B at home, under a box from FAI Orange
- Yunohost version 4.3.5
- I have access to my server: *Through SSH | through the webadmin (no problem)
- No special context, it has been installed several week ago

### Steps to reproduce
I had never seen any data to this part
I'd try several browser, and the problem occured on all of them : Vivaldi (5.0.2497.32), Edge (96.0.1054.62), Firefox (95.0.2) and Chrome (97.0.4692.71)

### Expected behavior
I have data, Aeneria collect data on my electric meter (such as it cne be seen on my screenshot)

### Logs
I have no error, so I don't know which log I can share with you...
You can see the problem on this screenshot
![image](https://user-images.githubusercontent.com/68971701/148139869-866e21ca-f2cf-4321-9a31-f636fd713607.png)

**Comment by SimonMellerin** [comment]
Hi,

In your Enedis space, did you accept hourly consumption collect in "Gérer l'accès à mes données" tab ?

**Comment by SebMidon** [comment]
Indeed, this notion was "deactivated" on my client interface. I don't understand since when it was disabled but that's certainly the explanation for it. I'll keep an eye on it in the next few days and keep you posted. If that's it, I'm sorry for create a ticket for a so little trouble...

**Comment by SimonMellerin** [comment]
No problem.

To understand, did you had this hourly data and then lost it ? Or has it always been like this ?

I should maybe add a notice while someone creates an Address to explain this

**Comment by SebMidon** [comment]
Thank you for your answers.
I recently reinstalled this app. But several months ago (over a year), I had already tested the old version of this service (Pilea). From this application, it worked. But without using their service for many months, the hourly collection had to turn off (I personally had not turned it off after I shut down Pilea). I suspect it was the amount of time without data collection that (possibly) disabled my permission to collect hourly data.
![image](https://user-images.githubusercontent.com/68971701/148617079-4e84933c-204a-4970-9856-55fa06baa5b4.png)
